### PR TITLE
Do not show change passphrase option when launching

### PR DESCRIPTION
**Pull request by guilhermemntt**
Closes https://github.com/decred/decrediton/issues/2790.

### CODE REVIEW

File: app/components/views/GetStartedPage/PrivacyPage/CustomPrivacyOptions.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default ({ toggleCustomPrivacy, acceptCustomPrivacy, ...props }) => (
       />
     </div>
     <div className={styles.customPrivacySettings}>
-      <PrivacySettings {...props} />
+      <PrivacySettings {...{ ...props, showChangePassPhraseOption: false }} />
```
Comment (vctt94) [inline app/components/views/GetStartedPage/PrivacyPage/CustomPrivacyOptions.jsx:20]: Thanks, but we already have a props which can be used for that. `walletReady`

**Review by vctt94 — COMMENTED** [review]
Thanks for this PR, I left a comment about a props which can be used for that, instead of adding a new one into the component.

File: app/components/views/GetStartedPage/PrivacyPage/CustomPrivacyOptions.jsx (hunk 1 above)
Comment (guilhermemntt) [reply app/components/views/GetStartedPage/PrivacyPage/CustomPrivacyOptions.jsx:20]: @vctt94 Great! That's good to know. I'll fix it.

**Comment by guilhermemntt** [comment]
It seems to be ok now, @vctt94.

**Review by vctt94 — APPROVED** [review]
Thanks, LGTM

**Pull request merged**